### ISSUE TITLE
false positive: unused import when the same name is used in a class' scope

### ISSUE THREAD

**Issue by afg984** [PyCQA/pylint#1330]
### Steps to reproduce
run pylint against the code:

```python
import struct

class MyClass:
    struct = struct.Struct('<II')
```

### Current behavior
```
W:  1, 0: Unused import struct (unused-import)
```

### Expected behavior
It is not unused

### pylint --version output
```
No config file found, using default configuration
pylint 2.0.0,
astroid 1.5.0
Python 3.6.0 (default, Jan 16 2017, 12:12:55)
[GCC 6.3.1 20170109]
```

**Comment by PCManticore** [comment]
Hi. Thanks for reporting this issue. Unfortunately, I cannot reproduce it using the latest code from 2.0 and astroid 1.5.0 respectively. Here's my output:

```
C:  1, 0: Missing module docstring (missing-docstring)
C:  3, 0: Missing class docstring (missing-docstring)
C:  3, 0: Old-style class defined. (old-style-class)
W:  3, 0: Class has no __init__ method (no-init)
R:  3, 0: Too few public methods (0/2) (too-few-public-methods)
```

Can you check with the latest code, please, and let me know if you still have this problem?

**Comment by afg984** [comment]
I can confirm it does not happen with latest default branch
